### ISSUE TITLE
Feature Request: Enable Dockerization for the Project

### ISSUE THREAD

**Issue by eyadfarra** [AIHawk-co/Auto_Jobs_Applier#509]
### Feature summary

Add Docker support to the project, allowing the application to run in a containerized environment for easier setup, development, and consistency.

### Feature description

Introduce Docker support by adding a Dockerfile and docker-compose.yml to create a development environment using Docker. This update will also include modifications to the README.md to guide users on building, running, and stopping the application using Docker.

### Motivation

Dockerizing the project simplifies the development setup by eliminating the need for a local Python installation. It ensures consistency across different environments, streamlines the onboarding process for new contributors, and makes dependency management more manageable.

### Alternatives considered

1. **Manual Local Setup**: Continue using the existing local installation setup. However, this approach requires users to install and configure Python and dependencies manually, leading to inconsistencies and potential version conflicts.

2. **Virtual Environments**: Using Python virtual environments (e.g., venv) can isolate dependencies, but it still requires manual setup and lacks the benefits of containerization (e.g., OS-level consistency).

**Comment by eyadfarra** [comment]
### Summary
I have created a branch with changes to support running the application on Docker instead of requiring a local Python installation. The updates include:
- Adding a Dockerfile and docker-compose.yml to set up the development environment.
- Modifying the README.md file to include instructions on how to build and run the application using Docker.

I have made local changes and created a new branch with the updates, but I am unable to push the new branch due to access restrictions. I would appreciate assistance in resolving this issue and guidance on how to proceed.

**Comment by LBuchananCates** [comment]
@eyadfarra That sounds amazing, can you fork this and share it in the event you can't push your branch? That'd solve so many of my problems.

**Comment by srtandon** [comment]
@eyadfarra Is there somewhere I can get the branch? I was just thinking I need a container for this. It would be super helpful.

**Comment by eyadfarra** [comment]
@LBuchananCates @srtandon 

Here’s the link to the PR:
[PR #585 - Add Docker Support](https://github.com/feder-cr/Auto_Jobs_Applier_AIHawk/pull/585)

**Comment by gustavosett** [comment]
@eyadfarra @LBuchananCates @srtandon 

I saw that the pull request was closed, but I was able to containerize it and make it work.

Now, to make it work completely, I need to overcome some challenges, such as finding a way to pass email verification even if two-step verification is disabled.

Any contribution is welcome.
[Containered_Auto_Jobs_Applier_AIHawk](https://github.com/gustavosett/Containered_Auto_Jobs_Applier_AIHawk)
